specs/iota-rs-ENGINEERING-SPEC-0000.md

```diff
@@ -49,90 +50,88 @@ The data structure to initialize the instance of the Higher level client library
 
 ## Sync Process
 
-When a `Client` instance (The instance which is used for calling the client APIs) is built, the status of each node listed in the `node_pool_urls` should be checked first. If the returned status of the node information is healthy, which means the node is synced, then this node will be pushed back into a `synced_node_list`. The rust-like pseudo code of `synced_node_list` construction process follows. The process of syncing a node is repeated every 60 seconds or at the interval specified in the `node_sync_interval` argument of the initializer if set.
+When a `Client` instance (The instance which is used for calling the client APIs) is built, the status of each node listed is checked. If the returned status of the node information is healthy, which means the node is synced, then this node will be pushed into a `synced_nodes` list. The rust-like pseudo code of `synced_nodes` construction process follows. The process of syncing a node is repeated every 60 seconds or at the interval specified in the `node_sync_interval` argument of the initializer if set.
 
 ```rust
-synced_node_list = Vec::new()
+synced_nodes = Vec::new()
 for node in node_pool_urls{
    status = Client.get_info(node).await?;
    if status == healthy{
-      synced_node_list.push(node)
+      synced_nodes.push(node)
    }
 }
 ```
 
-# General High level API
+# `General high level API`
 
 Here is the high level abstraction API collection with sensible default values for users easy to use.
 
-
 ## `send()`
 
-A generic send function for easily sending a value transaction message.
+A generic send function for easily sending a message.
 
 ### Parameters
 
-| Field | Required | Default | Type | Definition |
+| Parameter | Required | Default | Type | Definition |
 | - | - | - | - | - |
-| **seed** | ✘ | None | [Seed] | The seed of the account we are going to spend, only needed for SignedTransactions (value) |
-| **address** | ✘ | None | \[[Address]\] | The address(es) to send to, applies to value transactions only. |
-| **value** | ✘ | 0 | u64 | The amount of IOTA to send. If the value is zero the message object will have a IndexationPayload instead of a SignedTransactionPayload with an embedded IndexationPayload |
-| **path** | ✘ | `m/0'/0'` | [BIP32Path] | The wallet chain BIP32 path we want to search for. |
-| **output** | ✘ | None | \[Output\] | Users can manually pick their own output instead of having node decide on which output should be used. |
-| **indexation_key** | ✘ | None | String | An optional indexation key of the indexation payload. |
-| **data** | ✘ | None | [u8] | An optional indexation data of the indexation payload. |
-
-* If only `indexation_key` and `data` are provided. This method will create a message with only indexation payload instead.
+| **seed** | ✘ | None | [Seed] | The seed of the account we are going to spend, only needed for transactions |
+| **account_index** | ✘ | 0 | usize | The account index |
```

Comment:
Can we add an example of how the bip-path would look like for this for clarity in the spec? Same as for the next parameter.

Reply:
Where should I add this? Create an own section? Because it's used in multiple places